### ISSUE TITLE
A11y: In step-by-step mode every option is required

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5109]
When step-by-step mode is active, the custom product cannot be put into the cart if only one of the option was skipped. This is independent from the “required” checkbox, some of the options might have.

!image-20241014-144804.png|width=1548,height=1133,alt="image-20241014-144804.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-752**



[created from CUS-752, comment 503295]
